lib/src/base-styles/_nx-global-header.scss

```diff
@@ -6,22 +6,21 @@
  */
 @use '../scss-shared/nx-container-helpers';
 @use '../scss-shared/nx-button-helpers';
+@use '../scss-shared/nx-text-helpers';
 
 .nx-global-header {
   align-items: center;
-  border-bottom: var(--nx-border-default);
+  background-color: var(--nx-swatch-white);
   display: flex;
   grid-area: header; // see nx-page's grid-template
-  min-height: 32px;
-  padding: var(--nx-spacing-6x);
-
-  &:empty {
-    // if it has no content just don't show it
-    display: none;
-  }
+  height: 42px;
+  padding: 0 var(--nx-spacing-6x);
 
   .nx-back-button {
+    @include nx-text-helpers.truncate-ellipsis;
     margin: 0;
+    order: -1; // ensures back button is always on the left
+    padding-right: var(--nx-spacing-8x);
```

Comment:
Minor but IMO this should be margin.

Reply:
> Minor but this should be margin.

Changed here: 1b137a86439f5b0738a5ac2d0339fa4e8949c486